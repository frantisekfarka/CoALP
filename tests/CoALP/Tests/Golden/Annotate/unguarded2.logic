p(1).
p(X):-q(X),p2(X),q(X).
p2(X):-p(X).
