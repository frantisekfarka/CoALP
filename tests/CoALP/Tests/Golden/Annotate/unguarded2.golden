p(1, transform-func-1).
p(V_2, transform-func-2(V_4, V_-5, V_6)) :- q(V_2, V_4), p2(V_2, V_-5), q(V_2, V_6).
p2(V_3, transform-func-3(V_-7)) :- p(V_3, V_-7).
